kiwi/runtime_checker.py

```diff
@@ -190,25 +190,29 @@ def check_volume_setup_has_no_root_definition(self):
             if volume.mountpoint == '/':
                 raise KiwiRuntimeError(message)
 
-    def check_docker_tool_chain_installed(self):
+    def check_container_tool_chain_installed(self):
         """
-        When creating docker images the tools umoci and skopeo are used
-        in order to create docker compatible images. This check searches
-        for those tools to be installed in the build system and fails if
-        it can't find them
+        When creating container images the specific tools are used in order
+        to import and export OCI or Docker compatible images. This check
+        searches for those tools to be installed in the build system and
+        fails if it can't find them
         """
         message_tool_not_found = dedent('''\n
             Required tool {name} not found in caller environment
 
-            Creation of docker images requires the tools umoci and skopeo
-            to be installed on the build system. For SUSE based systems
+            Creation of OCI or Docker images requires the tools {name} and
+            skopeo to be installed on the build system. For SUSE based systems
             you can find the tools at:
 
             http://download.opensuse.org/repositories/Virtualization:/containers
         ''')
         message_version_unsupported = dedent('''\n
             {name} tool found with unknown version
         ''')
+        message_unknown_tool = dedent('''\n
+            Unknown tool:\'{0}\'. Please set \'oci_archive_tools\' with
+            \'umoci\' or \'buildah\' value'
+        ''')
```

Comment:
Sorry overlooked that one on the first review. I was wondering if that message quoting is needed ? I know to printout single ticks you need to quote them but it looks weird and actually we are suggesting to export a shell variable oci_archive_tools if I read that correctly. In shell people mostly use ```"```. Couldn't we make the error message more explicit and say e.g

```
message_unknown_tool = dedent('''\n
    Unknown tool: {0}. Please setup your environment as follows

    for use with umoci:
    $ export oci_archive_tools=umoci

    for use with buildah:
    $ export oci_archive_tools=buildah 
"'"')
```

or similar

Reply:
Ok, yes I agree to make it more explicit. What about now?